phpstan.neon.dist

```diff
@@ -5,5 +5,14 @@ parameters:
   scanFiles:
     - vendor/php-stubs/wordpress-stubs/wordpress-stubs.php
   treatPhpDocTypesAsCertain: false
+  ignoreErrors:
+    # TEMPORARY: WordPress 6.9 Abilities API functions not yet in stubs
+    - '#Function wp_get_ability not found\.#'
+    - '#Class WP_Ability not found\.#'
+    - '#on an unknown class WP_Ability\.#'
+    - '#has invalid type WP_Ability\.#'
+    -
+      message: '#expects .*, mixed given\.#'
+      path: includes/Builders/Prompt_Builder.php
```

Comment:
Maybe put a `TODO` here, or also cover in that issue for post-6.9 cleanup?

Reply:
We'll need to wait a little bit on this part since we'll need the https://github.com/php-stubs/wordpress-stubs library to update for 6.9.